lib/rules/jsx-curly-brace-presence.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @fileoverview Enforce curly braces or disallow unnecessary curly brace in JSX
+ * @author Jacky Ho
+ */
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+const optionAlways = 'always';
+const optionNever = 'never';
+const optionIgnore = 'ignore';
```

Comment:
Should these constants not be all upper case and use `_` as delimiter between words, to be more consistent with the rest of the code base? See `no-danger`, `no-typos`, `no-escaped-entities`, and others...

So: `OPTION_ALWAYS`, `OPTION_NEVER` and `OPTION_IGNORE`